### PR TITLE
update header styles

### PR DESCRIPTION
**Pull request by devoncarew**
- more work towards https://github.com/dart-lang/dartdoc/issues/1372
- borrow the header styles from dartlang.org
- two commits for easier reviewing

@jcollins-g 

screen shot 2017-05-12 at 3 23 58 pm

### CODE REVIEW

**Comment by devoncarew** [comment]
(I'll need to add some changelog entries for this work)

File: lib/src/html/template_data.dart [hunk 1]
```diff
@@ -56,12 +56,7 @@ abstract class TemplateData<T extends Documentable> {
 
   Iterable<Subnav> getSubNavItems() => <Subnav>[];
 
-  String _layoutTitle(String name, String kind, bool isDeprecated) {
-    if (kind.isEmpty) kind = '&nbsp;';
-    String str = '<span class="kind">$kind</span>';
-    if (!isDeprecated) return '${str} ${name}';
-    return '${str} <span class="deprecated">$name</span>';
-  }
+  String _layoutTitle(String name, String kind) => '${kind} ${name}';
```
Comment (jcollins-g) [inline lib/src/html/template_data.dart]: what does a page for a deprecated identifier look like now?

File: testing/test_package_docs/ex/Animal-class.html [hunk 2]
```diff
@@ -30,7 +30,7 @@
           <ol class="breadcrumbs gt-separated dark hidden-xs">
             <li><a href="index.html">test_package</a></li>
             <li><a href="ex/ex-library.html">ex</a></li>
-            <li class="self-crumb">Animal</li>
+            <li class="self-crumb">enum Animal</li>
```
Comment (jcollins-g) [inline testing/test_package_docs/ex/Animal-class.html]: I notice we used to use a span class for the kind but don't anymore, is that intentional?

Comment (devoncarew) [inline testing/test_package_docs/ex/Animal-class.html]: Yeah, the style for this became a no-op.

**Comment by devoncarew** [comment]
> It would be really helpful to have something browseable for this review so I can get a feel for how the new layout works in practice.

The easiest way might be to:
- git fetch origin
- git checkout css_header
- open open testing/test_package_docs/index.html

but I agree, it might be nice to have an easier way to have a staging preview server

File: lib/src/html/template_data.dart (hunk 1 above)
Comment (devoncarew) [inline lib/src/html/template_data.dart]: updated the PR; here's the latest:

screen shot 2017-05-15 at 7 50 49 pm

**Comment by jcollins-g** [comment]
lgtm

**Pull request merged**